Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN pip install --upgrade pip virtualenv
 # Install the Google-built interpreters
 ADD interpreters.tar.gz /
 
+# Add Google-built interpreters to the path
+ENV PATH $PATH:/opt/python3.5/bin
```

Comment:
Heh, in this case we built with `altinstall` so it should be safe to put this at the start of the path.